lock-api-objects/src/main/java/com/palantir/lock/watch/CommitUpdate.java

```diff
@@ -68,15 +79,22 @@ interface InvalidateSome extends CommitUpdate {
 
         Set<LockDescriptor> invalidatedLocks();
 
+        Map<LockDescriptor, List<ChangeMetadata>> aggregatedMetadata();
+
```

Comment:
Possibly need to think about naming here (the metadata is not really aggregated, just stuffed into a list)